doc/conf.py

```diff
@@ -0,0 +1,146 @@
+    "sphinx.ext.viewcode",
+    "sphinx.ext.intersphinx",
+    "sphinx_copybutton",
+    "sphinx_automodapi.automodapi",
+    "m2r2"
+]
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ['_templates']
+
+# List of patterns, relative to source directory, that match files and
+# directories to ignore when looking for source files.
+# This pattern also affects html_static_path and html_extra_path.
+exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store']
+
+# The master toctree document.
+master_doc = "index"
+
+autosummary_generate = True
+autosummary_imported_members = False
+
+# -- Options for HTML output -------------------------------------------------
+
+# The theme to use for HTML and HTML Help pages.  See the documentation for
+# a list of builtin themes.
+#
+html_theme = 'xanadu'
+
+# Add any paths that contain custom static files (such as style sheets) here,
+# relative to this directory. They are copied after the builtin static files,
+# so a file named "default.css" will overwrite the builtin "default.css".
+html_static_path = ['_static']
+
+# If true, sectionauthor and moduleauthor directives will be shown in the
+# output. They are ignored by default.
+show_authors = True
+
+# The name of the Pygments (syntax highlighting) style to use.
+pygments_style = "sphinx"
+
+# If true, `todo` and `todoList` produce output, else they produce nothing.
+todo_include_todos = False
+
+
+# The name of an image file (relative to this directory) to use as a favicon of
+# the docs.  This file should be a Windows icon file (.ico) being 16x16 or 32x32
+# pixels large.
+html_favicon = "_static/favicon.ico"
+
+# Custom sidebar templates, must be a dictionary that maps document names
+# to template names.
+html_sidebars = {
+    "**": [
+        "searchbox.html",
+        "globaltoc.html",
+    ]
+}
+
+
+# Xanadu theme options (see theme.conf for more information).
+html_theme_options = {
+    "navbar_name": "Xanadu Sphinx Theme",
+    "navbar_logo_colour": "#2d7c7f",
+    # "navbar_home_link": "https://pennylane.ai",
+    "navbar_left_links": [
+        {
+            "name": "Example1",
+            "href": "javascript:void(0);",
+        },
+        {
+            "name": "Blog",
+            "href": "javascript:void(0);",
+        },
+        {
+            "name": "Documentation",
+            "href": "index.html",
+            "active": True,
+        }
+    ],
+    "navbar_right_links": [
+        {
+            "name": "GitHub",
+            "href": "https://github.com/XanaduAI/xanadu-sphinx-theme",
+            "icon": "fab fa-github",
+        },
+    ],
+    "extra_copyrights": [
+        "Any extra copyrights can be added here"
+    ],
+    "border_colour": "#19b37b",
+    "prev_next_button_colour": "#19b37b",
+    "prev_next_button_hover_colour": "#0e714d",
+    "table_header_background_colour": "#edf7f4",
+    "text_accent_colour": "#19b37b",
+    "toc_marker_colour": "#19b37b",
```

Comment:
We should use the [official XST colour](https://material.io/resources/color/#!/?view.left=0&view.right=0&primary.color=F48FB1) to differentiate it from the other websites:
```suggestion
    "border_colour": "#f48fb1",
    "prev_next_button_colour": "#f48fb1",
    "prev_next_button_hover_colour": "#ec407a",
    "table_header_background_colour": "#f8bbd0",
    "text_accent_colour": "#f48fb1",
    "toc_marker_colour": "#f48fb1",
```

Reply:
Nice!